src/CodeGen.fs

```diff
@@ -923,113 +927,39 @@ type StringEnumAttribute() =
-    </ItemGroup>
-</Project>
-    """ attribute files
-
-let sampleSharedProject files =
-    sprintf """<Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <LangVersion>latest</LangVersion>
-    </PropertyGroup>
-    <ItemGroup>
-%s
-    </ItemGroup>
-    <ItemGroup>
-        <PackageReference Update="FSharp.Core" Version="4.7.2" />
-    </ItemGroup>
-</Project>
-"""  files
-
-let sampleSharedFSharpProject project copyLocalLockFileAssemblies =
-    let attribute =
-        match copyLocalLockFileAssemblies with
-        | None -> String.empty
-        | Some true -> "\n        <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>"
-        | Some false -> "\n        <CopyLocalLockFileAssemblies>false</CopyLocalLockFileAssemblies>"
-
-    sprintf """<Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <LangVersion>latest</LangVersion>%s
-    </PropertyGroup>
-    <ItemGroup>
-        <Compile Include="%s.GraphqlClient.fs" />
-    </ItemGroup>
-    <ItemGroup>
-        <PackageReference Update="FSharp.Core" Version="4.7.2" />
-        <PackageReference Include="Fable.Remoting.Json" Version="2.14.0" />
-        <ProjectReference Include="..\shared\%s.Shared.fsproj" />
-    </ItemGroup>
-</Project>
-"""  attribute project project
-
-let sampleSharedFableProject project =
-    sprintf """<Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <LangVersion>latest</LangVersion>
-    </PropertyGroup>
-    <ItemGroup>
-        <Compile Include="%s.GraphqlClient.fs" />
-    </ItemGroup>
-    <ItemGroup>
-        <Content Include="*.fsproj; *.fs; *.js" Exclude="**\*.fs.js" PackagePath="fable\" />
-    </ItemGroup>
-    <ItemGroup>
-        <PackageReference Update="FSharp.Core" Version="4.7.2" />
-        <PackageReference Include="Fable.SimpleHttp" Version="3.0.0" />
-        <PackageReference Include="Fable.SimpleJson" Version="3.19.0" />
-        <ProjectReference Include="..\shared\%s.Shared.fsproj" />
-    </ItemGroup>
-</Project>
-"""  project project
+let generateProjectDocument
+    (packageReferences: XElement seq)
+    (files: XElement seq)
+    (copyLocalLockFileAssemblies: bool option)
+    (contentItems: XElement seq)
+    (projectReferences: XElement seq)=
+    XDocument(
+        XElement.ofStringName("Project",
+            XAttribute.ofStringName("Sdk", "Microsoft.NET.Sdk"),
+            seq {
+            XElement.ofStringName("PropertyGroup",
+                seq {
+                    XElement.ofStringName("TargetFramework", "netstandard2.0")
+                    XElement.ofStringName("LangVersion", "latest")
+                    if copyLocalLockFileAssemblies.IsSome then
+                        XElement.ofStringName("CopyLocalLockFileAssemblies", copyLocalLockFileAssemblies.Value)
```

Comment:
I am not sure if this is a problem but in the previous version, `<CopyLocalLockFileAssemblies>` would have a lower-case value of `true` or `false`. Now when using `copyLocalLockFileAssemblies.Value`, the value of the boolean will be uppercase. Can you please make sure the old behaviour is retained? i.e. using `false` and `true` instead of `True` and `False`?

Reply:
ok, I'll edit this

Reply:
@anthony-mi, fixed?